src/Drivers/nlpMDS_raja_ex4.cpp

```diff
@@ -6,17 +6,9 @@
 #include <hiopMatrixDenseRowMajor.hpp>
 #include <hiopMatrixRajaDense.hpp>
 
-#ifdef HIOP_USE_GPU
-  #include "cuda.h"
-  #define RAJA_CUDA_BLOCK_SIZE 128
-  using ex4_raja_exec   = RAJA::cuda_exec<RAJA_CUDA_BLOCK_SIZE>;
-  using ex4_raja_reduce = RAJA::cuda_reduce;
-  #define RAJA_LAMBDA [=] __device__
-#else
-  using ex4_raja_exec   = RAJA::omp_parallel_for_exec;
-  using ex4_raja_reduce = RAJA::omp_reduce;
-  #define RAJA_LAMBDA [=]
-#endif
+#include <hiop_raja_defs.hpp>
```

Comment:
I don't think we should include `hiop_raja_defs.hpp` in the example. I suggest we keep this header out of HiOp API altogether. There is no reason why a user would want to use same policies as in HiOp library. User should be advised to use _compatible_ policies, but not necessarily the same. For example, for certain models different CUDA block size might be needed.